src/main/resources/data/miningmaster/recipes/forging/ultima_pickaxe_1.json

```diff
@@ -11,16 +11,19 @@
       "item": "miningmaster:lucky_citrine"
     },
     {
-      "item": "miningmaster:haste_peridot"
+      "item": "miningmaster:haste_peridot_block"
```

Comment:
Why does this one take a block, recipe 2 takes a single gem, and it looks like recipe 3 doesn't take any haste peridot?

Reply:
Recipe 2 takes three Peridots (which is still a mistake) whilst recipe 3 doesn't take any Peridots because the catalyst is a Peridot Pickaxe. I realise the block solution is a little clunky, so I've decided to rejig them again without the block.